src/main/java/teammates/ui/webapi/GetFeedbackSessionAction.java

```diff
@@ -71,4 +69,39 @@ public JsonResult execute() {
 
         return new JsonResult(response);
     }
+
+    private FeedbackSessionData getFeedbackSessionDataWithFilteredDeadlines(Intent intent) {
+        String courseId = getNonNullRequestParamValue(Const.ParamsNames.COURSE_ID);
+        String feedbackSessionName = getNonNullRequestParamValue(Const.ParamsNames.FEEDBACK_SESSION_NAME);
+        FeedbackSessionAttributes feedbackSessionAttributes = getNonNullFeedbackSession(feedbackSessionName, courseId);
+
+        FeedbackSessionData response;
+        switch (intent) {
+        case STUDENT_SUBMISSION:
+            // fall-through
+        case STUDENT_RESULT:
+            StudentAttributes studentAttributes = getStudentOfCourseFromRequest(courseId);
+            String studentEmailAddress = studentAttributes.getEmail();
```

Comment:
`studentEmailAddress` -> `studentEmail`? Doesn't sound like `Email` can be anything but the address.